python/tank/authentication/ui/resources_rc.py

```diff
@@ -8,9 +8,9 @@
x91\xfb>\x04\xa4\xaf\xa6\xba\x8f\x16\xbe>\xd5\xff\x8f\xdb]\xab(\x1e\x082\x95\xba\x1e\x97\x03\xdeLS\xb1(w\x1f\x1ed7\xd71\xc0\xc7\xea\xbcYk\xb1\x8d6\xc6\x95v\xf24\x19\xab\xa9&\xf37\x9f\xce*\x7f\xa3\xad\xf4q\xd1\xe51 d#\x8dx\xc6\xb3H\xfa\xb7\xa9%\x0b\x83\x9e\xa3S\xe4\x84<M\xe3\xa0\xcb%\xcb\x02\xdaGl\xea\xfbx%\xd8YDN\xe6j\xaa\xc7\xe4!\xda9\xca\xf9f\xc3x\xaa\xca\xc7AO\xdd\x1d\xe4\xb8<G-\xdc?P\xfc\x96&\xd2]\xc2q~\xed\x18YD-\xfa\xe2U\xec\xbc\xd1\x83B\xa6p\x03{#\x18e\x05\x0d\x18\xe9\xd1\x0c|#\x0b\xc7\x7f\xf9\x88*\x0c\xf2\xac\x14\xd3t\xae\x92N\x9eg\x18\xf1\x15Y'-i\xc5R\xd7:\xdcH'\xeaI(\xd29G\x8a\x1c\x93\xc1\x5c\xca\xcb\xc6UWMI\xe2Iz\x19\xea0\x97Z\x0ec)\x0e\xf2O\x1a\xc8\xfaP\x1c-FD\xd6E\x16\xf7K?.\xe2\x09\x97+\x81\x1c&\x91\xab\xe4\x06\xf1\xf6\xad6 d\x86\xd4\xa7%\xd3\x22>\x89_LG\xaa\xc8\xf8\xe8=\xa4<\x1b\xf9U\x1e\xa7<\xfd\xd9\xeaR\x87s\xb8\x8b\x0a2P\xf6\x1ak\xb0\x85\xa6tg\x97\xd5(\xfby\x95J\xf2\x82\x1c\x079\xc8\xef>.\x98\x07d\x1bm${\xf97\xff\xd6\x1a\xdcNK\xeaF\xe8\xdd\x9d\xcc,&\xf3\xa9\xf9\x17\x13\x9d\xc8Lfj%:s\xb7qn\xf6\xf4\xecd\x02\x89\xb2<\xe8Yx\x83\xfc\xc6\xf3\xfa\x227\xd0\x916\x8e+\xc9\xa7\xb2\x90\x8f\xf8\xc0I\xda\x0dI\xe1m}\x8f\xce<L5\x83q\x163\x8eq\x19\xd2um\xe1\x82@\x16\xce%\x8c\xc2O\xb40\x0d\xa9E\x0d\xaas\x99q\xednH\xe6GV\xb2\x82E\xac\xb0\xbf\x1e\xd2\xd2\x146\x10\xfb\xdd\xfc-V\x8bR\xd2@\xec\x88\xb8\xe0\x0e\xac5\xb9\x91\xa642Z\xad\xc3,b\x01_\xb0\xc8\xdb\xa7\xb8\xc6\x9a\xa5\x92\x96\xcd^j\x01Z\x88\xa6\xb4\xa29\xb5\xc9o\xd8\xe48\xab\xf9\x86\xb9\xcc\x11\x17\x9e\xabZ\x8d64\xa1~\x16\xf9\x94\xf6\xb2\x8e\xe5,`\xf6\xd9\x11\xffZ\x96?\xe5\xdau\x8e\xbf\x1d\xbd\xc8(\x887I\x8c\x93gk\x0c\xe5M\xe4\xeam64\xf4\x0c\x9d\x17\xe1b.\xe6/\xc4S\x9cx\xe2\x808\x0a\x01\xa9\xec\x03\xf6\xb2\x9f\xbd\xec`\x17\x9b\xa3'P\xc5+4\x86+\xa9C%*q\x19e)Ba\x0a\x03G9D2\xbf\xb1\x89\x1f\xd9\xc8j\x96E\xf7%\xa3\xe3\xd5\x89\xa3\x16U\xa9\xc8\xa5T\xa0\x1cq\xc4\x9d.\x91\xb4\x8f\x83\xecf7\xdb\xd9\xc0&\xd6\xb0Z\x8e{0~\x09*R\x8cx\xe0\x04\xc9\xfc\xcaN\xc9\xa3w\xee\xa9@\xbe\xa8\xbf\xe1:\xcfy\xce\x93+\xff\x05cMD\xa0\xca\x19\x84H\x00\x00\x00%tEXtdate:create\x002018-01-08T10:05:17-05:00\xf4ga\xd5\x00\x00\x00%tEXtdate:modify\x002018-01-08T10:05:17-05:00\x85:\xd9i\x00\x00\x00\x19tEXtSoftware\x00Adobe ImageReadyq\xc9e<\x00\x00\x00\x00IEND\xaeB`\x82\x00\x00\x08\x90\x89PNG\x0d\x0a\x1a\x0a\x00\x00\x00\x0dIHDR\x00\x00\x002\x00\x00\x00f\x08\x03\x00\x00\x00\xbe\xfe2b\x00\x00\x00\x04gAMA\x00\x00\xb1\x8f\x0b\xfca\x05\x00\x00\x00 cHRM\x00\x00z&\x00\x00\x80\x84\x00\x00\xfa\x00\x00\x00\x80\xe8\x00\x00u0\x00\x00\xea`\x00\x00:\x98\x00\x00\x17p\x9c\xbaQ<\x00\x00\x02\xd3PLTE\x00\x00\x00\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xfb\xfb\xfb\xf9\xf9\xf9\xf8\xf9\xf9\xf9\xf9\xfb\xfd\xfd\xfd\xfa\xfa\xfa\xfb\xfb\xfb\xf2\xf2\xf3\xd2\xd2\xd4\xfa\xfa\xfa\xff\xff\xff\xff\xff\xff\xf9\xf9\xf9\xcb\xcb\xcd\xf2\xf2\xf2\xf9\xf9\xfa\xff\xff\xff\xf9\xf9\xfb\xf8\xf8\xf8\xc4\xc4\xc7\xf7\xf8\xf8\xfa
```

Comment:
I've updated `build_resources.sh` to add a flag to the pyside-rcc command `-py3` -- this add the `b` prefix to the string.  I believe (and testing seems to confirm) that the resource compilation from pyside1 is compatible with pyside2, and the `b` prefix is ignored in py2.7+, however will be incompatible with 2.6 and below, and 3.4 and below I believe.  Given our target python versions I deemed this acceptable.  I didn't document new build procedures since the build procedure is unchanged (the modifications to `build_resources.sh` take care of everything.

To generate these files I did rebuild, although for the sake of cleaning up the diff I removed the timestamp changes.  I can put them back in if you think that was a mistake.

Reply:
Hum, what timestamp changes? I thought that   `sed -i "" -e "s/from PySide import/from . import/g" -e "/# Created:/d" $UI_PYTHON_PATH/$3.py` took out the timestamp for that reason, to avoid any timestamp shenanigans.

Reply:
I think there are some timestamps embedded in the resources themselves.  It could be fine to just commit the change, I don't think it would affect anything.  I suspect the timestamp change stems from running `mogrify` on the images, probably updating the modified timestamp at that point.

<img width="1127" alt="Screen Shot 2019-10-04 at 8 45 30 AM" src="https://user-images.githubusercontent.com/5922577/66221366-b0091400-e683-11e9-909a-85e0631c799b.png">

Reply:
Actually, on second thought, looking at that timestamp I don't know what it is...  It's definitely not today's date.